src/status_im2/contexts/chat/messages/content/view.cljs

```diff
@@ -46,61 +47,70 @@
            quoted-message
            from
            timestamp]}
-   message-reaction-view?]
-  (when (or (and (seq response-to) quoted-message) last-in-group? pinned-by message-reaction-view?)
+   show-reactions?]
+  (when (or (and (seq response-to) quoted-message) last-in-group? pinned-by (not show-reactions?))
     (let [[primary-name secondary-name] (rf/sub [:contacts/contact-two-names-by-identity from])
           {:keys [ens-verified added?]} (rf/sub [:contacts/contact-by-address from])]
-      [quo/author
-       {:primary-name   primary-name
-        :secondary-name secondary-name
-        :short-chat-key (address/get-shortened-key (or compressed-key from))
-        :time-str       (datetime/timestamp->time timestamp)
-        :contact?       added?
-        :verified?      ens-verified}])))
+      [rn/view {:margin-bottom 8}
```

Comment:
Missing `:style`

Reply:
It's not needed for `rn/view`

Reply:
Sorry, I wasn't clear enough. I mean wrapping `{:margin-bottom 8}` inside `{:style ...}`, as per the guidelines.